tor/core/user_interaction.py

```diff
@@ -53,10 +53,10 @@ def process_coc(post, config):
             f'<http://www.reddit.com/u/{post.author.name}|u/{post.author.name}>'
             f' has just accepted the CoC!', config
         )
-    process_claim(post, config)
+    process_claim(post, config, first_claim=True)
```

Comment:
I like that we’re setting this as a keyword argument instead of a positional one. Definitely reveals the intention a lot more clearly. :+1: